src/reducers/saleCart.js

```diff
@@ -0,0 +1,9 @@
+let initialState = {
+    items: []
+};
+export default function saleCartReducer(state = initialState, action) {
+  switch(action.type) {
+    default:
+      return state;
+  }
+}
```

Comment:
Faltou uma linha no final do arquivo. Tem configuração do VS Code que ajuda nisso. Tbm tem uma extensão chamada final-newline. É só questão de [boas práticas](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) mesmo.